df/plugin.go

```diff
@@ -249,9 +344,13 @@ func Meta() *plugin.PluginMeta {
 }
```

Comment:
The option `plugin.RoutingStrategy(plugin.StickyRouting)` should be added to ensure that there is a 1:1 relationship between running tasks and running plugin instances.